archetype/simple-service/src/main/resources/archetype-resources/pom.xml

```diff
@@ -82,6 +82,11 @@
                 <version>2.0.2</version>
             </dependency>
             <dependency>
+                <groupId>com.google.guava</groupId>
+                <artifactId>guava</artifactId>
+                <version>15.0</version>
```

Comment:
This matches the version in the beadledom parent pom. Should we up that version too?

Currently beadledom-client and libraries shared between client and server are Java 6 compatible, so if we did bump the guava version we can only go as far as the lateast one supporting Java 6.